src/adapters/zns/execution/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "zklink",
```

Comment:
@xsteadybcgo We can change this template "zklink" to "zns" obviously? CC: @formysister

Reply:
zklink ==> zns